source/MaterialXGenOgsFx/OgsFxShaderGenerator.cpp

```diff
@@ -17,46 +16,46 @@ namespace
     // Semantics used by OgsFx
     static const StringMap OGSFX_DEFAULT_SEMANTICS_MAP =
     {
-        { "i_position", "POSITION"},
-        { "i_normal", "NORMAL" },
-        { "i_tangent", "TANGENT" },
-        { "i_bitangent", "BINORMAL" },
-
-        { "i_texcoord_0", "TEXCOORD0" },
-        { "i_texcoord_1", "TEXCOORD1" },
-        { "i_texcoord_2", "TEXCOORD2" },
-        { "i_texcoord_3", "TEXCOORD3" },
-        { "i_texcoord_4", "TEXCOORD4" },
-        { "i_texcoord_5", "TEXCOORD5" },
-        { "i_texcoord_6", "TEXCOORD6" },
-        { "i_texcoord_7", "TEXCOORD7" },
-
-        { "i_color_0", "COLOR0" },
-
-        { "u_worldMatrix", "World" },
-        { "u_worldInverseMatrix", "WorldInverse" },
-        { "u_worldTransposeMatrix", "WorldTranspose" },
-        { "u_worldInverseTransposeMatrix", "WorldInverseTranspose" },
-
-        { "u_viewMatrix", "View" },
-        { "u_viewInverseMatrix", "ViewInverse" },
-        { "u_viewTransposeMatrix", "ViewTranspose" },
-        { "u_viewInverseTransposeMatrix", "ViewInverseTranspose" },
-
-        { "u_projectionMatrix", "Projection" },
-        { "u_projectionInverseMatrix", "ProjectionInverse" },
-        { "u_projectionTransposeMatrix", "ProjectionTranspose" },
-        { "u_projectionInverseTransposeMatrix", "ProjectionInverseTranspose" },
-
-        { "u_worldViewMatrix", "WorldView" },
-        { "u_viewProjectionMatrix", "ViewProjection" },
-        { "u_worldViewProjectionMatrix", "WorldViewProjection" },
-
-        { "u_viewDirection", "ViewDirection" },
-        { "u_viewPosition", "WorldCameraPosition" },
-
-        { "u_frame", "Frame" },
-        { "u_time", "Time" }
+        { HW::T_IN_POSITION, "POSITION"},
```

Comment:
The semantics are applied during code generation (before token substitution), so names here should use the token names.